test/express-tests.test.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+var path = require('path'),
+  helpers = require('yeoman-generator').test,
+  assert = require('yeoman-generator').assert,
+  temp = require('temp').track();
+
+describe('Express Sub Generators Tests', function () {
+  this.timeout(0);
```

Comment:
@UndeadBaneGitHub I merged a PR that adds support for a global timeout in a better way. In a completely separate PR, just add "timeout: 0" here https://github.com/meanjs/generator-meanjs/blob/master/gulpfile.js#L30 and remove it from all the tests. That way we can just do it once. Let me know when you make that change and then I'll look over the PRs again, but they should be good to go so that should be the last change.